src/db/delete.rs

```diff
@@ -105,6 +104,7 @@ fn delete_crate_from_database(conn: &Connection, name: &str, crate_id: i32) -> R
             &[&crate_id],
         )?;
     }
+    transaction.execute("DELETE FROM owner_rels WHERE cid = $1;", &[&crate_id])?;
```

Comment:
I was going to say you only do this for deleting a crate, not a version, then I realized that's the right behavior and it was just buggy before 😅